src/ring_theory/algebraic.lean

```diff
@@ -128,3 +128,72 @@ begin
   refine ⟨⟨_, x_integral⟩, ⟨_, y_integral⟩, _, rfl⟩,
   exact λ h, a_ne_zero (inj _ (subtype.ext_iff_val.mp h))
 end
+section field
+
+variables {K L : Type*} [field K] [field L] [algebra K L] (A : subalgebra K L)
+
+lemma inv_eq_of_aeval_div_X_ne_zero {x : L} {p : polynomial K}
```

Comment:
These changes were part of another PR right? Do you need to merge master?

Reply:
Yeah, I rebased on top of the other branch, but didn't merge master because I forgot that bors squashes its merges.